### PR TITLE
Change references to "master" to "main".

### PR DESCRIPTION
**Pull request by cary-ilm**
* fix urls
* fix references in docs
* rename C++ variables
* remove "master_doc" from conf.py entirely since it's not necessary

Signed-off-by: Cary Phillips <cary@ilm.com>

### CODE REVIEW

File: bazel/third_party/zlib.BUILD [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copied from https://github.com/protocolbuffers/protobuf/blob/master/third_party/zlib.BUILD
+# Copied from https://github.com/protocolbuffers/protobuf/blob/main/third_party/zlib.BUILD
```
Comment (kdt3rd) [inline bazel/third_party/zlib.BUILD]: This looks like it is still master in the upstream repo, or did I miss read?

**Review by kdt3rd — CHANGES_REQUESTED** [review]
Everything else looks good, just the protocolbuffers repo looks like it is still using the same branch name. Do we even need that full url in that comment, or would https://github.com/protocolbuffers/protobuf suffice?

File: bazel/third_party/zlib.BUILD (hunk 1 above)
Comment (lgritz) [reply bazel/third_party/zlib.BUILD]: You're right, I think this was just over-zealous cut-and-paste.

Comment (cary-ilm) [reply bazel/third_party/zlib.BUILD]: Oops. Fixed now.

**Pull request merged**